code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -107,6 +107,8 @@
 					L.Weaken(5)
 				sleep(2)//Runtime prevention (infinite bump() calls on hulks)
 				step_towards(src,L)
+			else
+				Weaken(2)

```

Comment:
Make sure this actually works; I suspect this will actually do nothing; notice how weaken is set, var wise, directly, below.

Once the refactor for status procs is done/fixed, then I believe you can set this to be a *forced* update which will work, but for now, I don't believe this will work.